api/models/event.go

```diff
@@ -0,0 +1,10 @@
+package models
+
+type LogMessage struct {
+	Type      string `json:"type"`
+	EventTime int64  `json:"time"`
+	IsKnown   string `json:"isKnown"`
+	Username  string `json:"username"`
+	RFID      string `json:"uid"`
```

Comment:
why change the json to say `uid` instead of `rfid`? RFID makes more sense to me than UID

Reply:
This has to be UID because it's for unmarshaling the json from the rfid device.